src/Data/Integer/Properties.agda

```diff
@@ -944,8 +944,8 @@ distribʳ-⊖-+-neg m n o = begin
   ; ⁻¹-cong  = cong (-_)
   }
 
-+-isAbelianGroup : IsAbelianGroup _+_ +0 (-_)
-+-isAbelianGroup = record
++-0-isAbelianGroup : IsAbelianGroup _+_ +0 (-_)
++-0-isAbelianGroup = record
```

Comment:
We should add a deprecation warning for the old name at the bottom of the file.

Reply:
Aaargh... yes. 🤦